Foundation/FileManager.swift

```diff
@@ -2377,46 +2377,122 @@ extension FileManager {
             self.innerEnumerator = ie
         }
         
-        @available(Windows, deprecated, message: "Not Yet Implemented")
         override func nextObject() -> Any? {
             let o = innerEnumerator.nextObject()
             guard let url = o as? URL else {
                 return nil
             }
 
 #if os(Windows)
-            NSUnimplemented()
+            var relativePath: UnsafeMutableBufferPointer<WCHAR> = UnsafeMutableBufferPointer<WCHAR>.allocate(capacity: Int(MAX_PATH))
+            defer { relativePath.deallocate() }
+            guard TRUE == baseURL.absoluteString.withCString(encodedAs: UTF16.self, { pszFrom in
```

Comment:
Hmm, wouldn't we want this to be the FSR?  CC @millenomi

Reply:
Alright, prettied up  the PathRelativeToPathW call some what and used FSR now.